<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-DRJTCK0" failures="0" tests="0" name="Default test" time="0.222" errors="0" timestamp="2025-07-30T17:42:43 GMT-07:00">
  <testcase classname="Base_Class.Base_Class" name="@BeforeSuite connectToDatabase" time="1.25">
    <failure type="java.sql.SQLSyntaxErrorException" message="Unknown database &amp;apos;project&amp;apos;">
      <![CDATA[java.sql.SQLSyntaxErrorException: Unknown database 'project'
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:815)
at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:438)
at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241)
at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:189)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
at GenericUtilities.DatabaseUtility.getDatabaseConnection(DatabaseUtility.java:74)
at Base_Class.Base_Class.connectToDatabase(Base_Class.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- @BeforeSuite connectToDatabase -->
  <testcase classname="Base_Class.Base_Class" name="@BeforeTest conParallel" time="-1.753922563646E9">
    <skipped/>
  </testcase> <!-- @BeforeTest conParallel -->
  <testcase classname="Base_Class.Base_Class" name="@BeforeClass launchBrowser" time="-1.753922563741E9">
    <skipped/>
  </testcase> <!-- @BeforeClass launchBrowser -->
  <testcase classname="Base_Class.Base_Class" name="@BeforeMethod login" time="-1.753922563785E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod login -->
  <testcase classname="Base_Class.Base_Class" name="@AfterClass closeBrowser" time="-1.753922563811E9">
    <skipped/>
  </testcase> <!-- @AfterClass closeBrowser -->
  <testcase classname="Base_Class.Base_Class" name="@AfterTest closeConParallel" time="-1.753922563828E9">
    <skipped/>
  </testcase> <!-- @AfterTest closeConParallel -->
  <testcase classname="Organization_Module.CreateOrgScenerios" name="createOrgWithPhoneNum" time="0.0">
    <ignored/>
  </testcase> <!-- createOrgWithPhoneNum -->
  <testcase classname="Organization_Module.CreateOrgScenerios" name="createOrgWithIndustryType" time="0.0">
    <ignored/>
  </testcase> <!-- createOrgWithIndustryType -->
</testsuite> <!-- Default test -->
